src/ui/DynamicFacetManager/DynamicFacetManager.ts

```diff
@@ -159,7 +151,14 @@ export class DynamicFacetManager extends Component {
 
   private handleAfterComponentsInitialization() {
     this.childrenFacets = this.allDynamicFacets;
-    this.childrenFacets.forEach(dynamicFacet => (dynamicFacet.dynamicFacetManager = this));
+    this.childrenFacets.forEach(dynamicFacet => {
+      dynamicFacet.dynamicFacetManager = this;
+      this.containerElement.appendChild(dynamicFacet.element);
+    });
+
+    if (this.element.children.length > 1) {
```

Comment:
I'm missing context on why all children of the element incompatible. I see on line 128 that we prepend a `containerElement` for example.

Reply:
we move all compatible components inside the containerElement, this is then the only children normally, hence the check `> 1` any more than this would be non DNE stuff